scripts/get-operator-status.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -x
+
+if [ -z "${OPERATOR_NAMESPACE}" ]; then
+    echo "Please set OPERATOR_NAMESPACE"; exit 1
+fi
+
+if [ -z "$OPERATOR_NAME" ]; then
+    echo "Please set OPERATOR_NAME"; exit 1
+fi
+
+CSVNAME=$(oc get csv -n ${OPERATOR_NAMESPACE} -o jsonpath='{range .items[*]}{@.metadata.name}{"\n"}{end}' | egrep -e "^${OPERATOR_NAME}-operator\.v")
+if [ -z "$CSVNAME" ]; then
+    echo "NOTFOUND"
+    exit 1
+fi
```

Comment:
This `egrep -e "^${OPERATOR_NAME}-operator\.v"` doesn't seem to be working in my local env for Rabbit, because the operator CSV name is actually `rabbitmq-cluster-operator`